livre/core/render/GLContext.h

```diff
@@ -33,7 +33,7 @@ namespace livre
 class GLContext
 {
 public:
-    LIVRECORE_API GLContext();
+    LIVRECORE_API GLContext( const GLEWContext* glewContext );
```

Comment:
GLContext needs a GLEWContext, otherwise gl functions won't work. This is the point of this PR.

Where do we need it without a GLEWContext? How would this work?